src/types/transaction.zig

```diff
@@ -68,6 +71,132 @@ pub const Transaction = struct {
     lock_time: u32,
```

Comment:
```
inputs: std.ArrayList(Input),
outputs: std.ArrayList(Output),
```

Try with input and output being slices for not

Reply:
Same as for block. Should I do it anyway ?

Reply:
yup I think it should be possible